airflow/providers/microsoft/azure/hooks/asb.py

```diff
@@ -326,5 +336,27 @@ def receive_subscription_message(
                 max_message_count=max_message_count, max_wait_time=max_wait_time
             )
             for msg in received_msgs:
-                self.log.info(msg)
-                subscription_receiver.complete_message(msg)
+                self._process_message(msg, message_callback, subscription_receiver)
+
+    def _process_message(self, msg, message_callback, receiver):
```

Comment:
Could we add type annotation here? Thanks!

Reply:
yes, added.